src/routes.js

```diff
@@ -3,14 +3,19 @@ const {
     verifyOpenIDToken,
     verifyRoomMembership,
 } = require('./verify');
+const logger = require('./logger');
+const {requestLogger} = require('./utils');
 
 const routes = {
     getHealth: (req, res) => {
+        requestLogger(req);
         res.send('👍');
     },
     postVerifyUser: async(req, res) => {
+        requestLogger(req);
         const checkResult = sanityCheckRequest(req, res, ['token']);
         if (!checkResult) {
+            logger.log('info', 'Request sanity check failed.');
```

Comment:
Thought: Would be good to have requestIDs, like in Synapse so you can follow along a single request in Kibana.

Reply:
Great idea, added!

Reply:
```
{
  requestId: '40e68649-4723-4401-b890-557d978ae309',
  level: 'info',
  message: 'POST /verify/user: {"token":"foobar"}',
  timestamp: '2020-09-15T09:17:39.730Z'
}
{
  requestId: '40e68649-4723-4401-b890-557d978ae309',
  level: 'debug',
  message: 'Request sanity check ok',
  timestamp: '2020-09-15T09:17:39.731Z'
}
{
  requestId: '40e68649-4723-4401-b890-557d978ae309',
  level: 'debug',
  message: 'Making request to: https://federator.modular.im/_matrix/federation/v1/openid/userinfo?access_token=redacted',
  timestamp: '2020-09-15T09:17:39.731Z'
}
{
  requestId: '40e68649-4723-4401-b890-557d978ae309',
  level: 'debug',
  message: 'Verify token failed: 401, {"date":"Tue, 15 Sep 2020 09:17:39 GMT","content-type":"application/json","content-length":"71","connection":"close","cache-control":"no-cache, no-store, must-revalidate","access-control-allow-origin":"*","access-control-allow-methods":"GET, HEAD, POST, PUT, DELETE, OPTIONS","access-control-allow-headers":"Origin, X-Requested-With, Content-Type, Accept, Authorization","strict-transport-security":"max-age=15724800; includeSubDomains"}, {"errcode":"M_UNKNOWN_TOKEN","error":"Access Token unknown or expired"}',
  timestamp: '2020-09-15T09:17:39.999Z'
}
{
  requestId: '40e68649-4723-4401-b890-557d978ae309',
  level: 'info',
  message: 'User token check failed.',
  timestamp: '2020-09-15T09:17:40.000Z'
}



```

Reply:
Also now redacting the token from the POST logs.

Reply:
ooo great :)